Source/SwiftLintCore/Helpers/Stack.swift

```diff
@@ -3,7 +3,7 @@ public struct Stack<Element> {
     private var elements = [Element]()
 
     /// Creates an empty `Stack`.
-    public init() {}
+    public init() { /* Publish no-op initializer */ }
```

Comment:
If this is a legitimate exception, because it's making the constructor public, could we detect and ignore it. I think this would only apply to `init` as far as I can think of.